packages/manager/modules/pci/src/projects/project/storages/databases/database/metrics/metrics.controller.js

```diff
@@ -126,4 +139,14 @@ export default class {
       }, CHART_METRICS_REFRESH_INTERVAL);
     }
   }
+
+  static setMomentLocale(lang) {
+    let language;
+    if (['en_GB', 'es_US', 'fr_CA'].includes(lang)) {
+      language = lang.toLowerCase().replace('_', '-');
+    } else {
+      [language] = lang.split('_');
+    }
+    moment.locale(language);
```

Comment:
Already done here: https://github.com/ovh/manager/blob/2a224502775351c1c76d17a881c288ff1ddda84b/packages/manager/apps/public-cloud/src/app.module.js#L107

Reply:
Hi @antleblanc, the set a moment's locale in manager/packages/manager/apps/public-cloud/src/app.module.js does not seems to work, as the charts always display English format no matter what language the user set. I am not a 100% sure why, but calling moment.locale once again just before the chart creation fixes this issue, maybe it is because the chartjs directive is in manager-components folder, outside of apps/public-cloud ?
I agree with you that this should not be done in the controller creating the chart, hence I moved it to the chart directive, so it should be cleaner.